projects/rocprofiler-systems/source/bin/rocprof-sys-sample/impl.cpp

```diff
@@ -130,6 +129,52 @@ get_initial_environment()
     update_env(_env, "LD_LIBRARY_PATH", tim::filepath::dirname(_dl_libpath), UPD_APPEND);
     update_env(_env, "ROCPROFSYS_SCRIPT_PATH", _libexecpath, UPD_REPLACE);
 
+    // libomptarget discovery
+    auto rocm_dir  = get_env<std::string>("ROCM_PATH", "/opt/rocm", false);
+    auto rocmv_dir = get_env<std::string>("ROCmVersion_DIR", "", false);
+
+    // strip one trailing slash if present
+    auto strip = [](std::string s) {
+        if(!s.empty() && s.back() == '/') s.pop_back();
+        return s;
+    };
+    rocm_dir  = strip(rocm_dir);
+    rocmv_dir = strip(rocmv_dir);
+
+    // build candidate libdirs
+    std::vector<std::string> llvm_candidates = {
+        rocmv_dir.empty() ? std::string{} : rocmv_dir + "/llvm/lib",
+        rocmv_dir.empty() ? std::string{} : rocmv_dir + "/lib",
```

Comment:
```suggestion
        rocmv_dir + "/llvm/lib",
        rocmv_dir + "/lib",
```
There is no difference if rocmv_dir is empty or not.

Reply:
done